packages/experience-builder-sdk/rollup.config.mjs

```diff
@@ -37,5 +37,6 @@ export default [
     input: 'src/index.ts',
     output: [{ file: 'dist/index.d.ts', format: 'es' }],
     plugins: [dts({ compilerOptions: { noEmitOnError: process.env.DEV ? false : true } })],
+    external: [/@contentful\/experiences-core/],
```

Comment:
Importing from `core/types` in the SDK raises the warning that they are not added to the final build. In the `components` package, I decided to refactor all cases but this would is the quick solution for now to solve it through the rollup config.